src/Illuminate/Auth/CreatesUserProviders.php

```diff
@@ -48,11 +48,11 @@ public function createUserProvider($provider = null)
      * @param  string|null  $provider
      * @return array|null
```

Comment:
It seems the return value can't be `null` anymore, right?

Reply:
Not exactly. It will return `null` if given provider doesn't exist.